[2024-01-31T09:56:27.130-0300] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: model_adventureworks_sales_category.dbt_report scheduled__2024-01-30T00:00:00+00:00 [queued]>
[2024-01-31T09:56:27.141-0300] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: model_adventureworks_sales_category.dbt_report scheduled__2024-01-30T00:00:00+00:00 [queued]>
[2024-01-31T09:56:27.141-0300] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2024-01-31T09:56:27.161-0300] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): dbt_report> on 2024-01-30 00:00:00+00:00
[2024-01-31T09:56:27.166-0300] {standard_task_runner.py:60} INFO - Started process 16918 to run task
[2024-01-31T09:56:27.170-0300] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'model_adventureworks_sales_category', 'dbt_report', 'scheduled__2024-01-30T00:00:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/model_sales_category.py', '--cfg-path', '/tmp/tmpyv_h3q0u']
[2024-01-31T09:56:27.172-0300] {standard_task_runner.py:88} INFO - Job 45: Subtask dbt_report
[2024-01-31T09:56:27.211-0300] {task_command.py:423} INFO - Running <TaskInstance: model_adventureworks_sales_category.dbt_report scheduled__2024-01-30T00:00:00+00:00 [running]> on host LAPTOP-MELTBPNS.
[2024-01-31T09:56:27.273-0300] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='soda_core' AIRFLOW_CTX_DAG_ID='model_adventureworks_sales_category' AIRFLOW_CTX_TASK_ID='dbt_report' AIRFLOW_CTX_EXECUTION_DATE='2024-01-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-30T00:00:00+00:00'
[2024-01-31T09:56:27.275-0300] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-01-31T09:56:27.275-0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt run --project-dir /home/user/projects/newproj/soda//dbt/partner_cert --select report']
[2024-01-31T09:56:27.284-0300] {subprocess.py:86} INFO - Output:
[2024-01-31T09:56:30.590-0300] {subprocess.py:93} INFO - [0m12:56:30  Running with dbt=1.7.4
[2024-01-31T09:56:30.958-0300] {subprocess.py:93} INFO - [0m12:56:30  Registered adapter: postgres=1.7.4
[2024-01-31T09:56:31.114-0300] {subprocess.py:93} INFO - [0m12:56:31  Found 3 models, 4 sources, 0 exposures, 0 metrics, 401 macros, 0 groups, 0 semantic models
[2024-01-31T09:56:31.122-0300] {subprocess.py:93} INFO - [0m12:56:31
[2024-01-31T09:56:31.266-0300] {subprocess.py:93} INFO - [0m12:56:31  Concurrency: 1 threads (target='dev')
[2024-01-31T09:56:31.267-0300] {subprocess.py:93} INFO - [0m12:56:31
[2024-01-31T09:56:31.270-0300] {subprocess.py:93} INFO - [0m12:56:31  1 of 2 START sql view model public.report_category_sales ....................... [RUN]
[2024-01-31T09:56:31.423-0300] {subprocess.py:93} INFO - [0m12:56:31  1 of 2 OK created sql view model public.report_category_sales .................. [[32mCREATE VIEW[0m in 0.15s]
[2024-01-31T09:56:31.426-0300] {subprocess.py:93} INFO - [0m12:56:31  2 of 2 START sql view model public.report_subcategory_sales .................... [RUN]
[2024-01-31T09:56:31.495-0300] {subprocess.py:93} INFO - [0m12:56:31  2 of 2 OK created sql view model public.report_subcategory_sales ............... [[32mCREATE VIEW[0m in 0.07s]
[2024-01-31T09:56:31.515-0300] {subprocess.py:93} INFO - [0m12:56:31
[2024-01-31T09:56:31.516-0300] {subprocess.py:93} INFO - [0m12:56:31  Finished running 2 view models in 0 hours 0 minutes and 0.39 seconds (0.39s).
[2024-01-31T09:56:31.527-0300] {subprocess.py:93} INFO - [0m12:56:31
[2024-01-31T09:56:31.527-0300] {subprocess.py:93} INFO - [0m12:56:31  [32mCompleted successfully[0m
[2024-01-31T09:56:31.528-0300] {subprocess.py:93} INFO - [0m12:56:31
[2024-01-31T09:56:31.528-0300] {subprocess.py:93} INFO - [0m12:56:31  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-01-31T09:56:32.782-0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-31T09:56:32.798-0300] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=model_adventureworks_sales_category, task_id=dbt_report, execution_date=20240130T000000, start_date=20240131T125627, end_date=20240131T125632
[2024-01-31T09:56:32.839-0300] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-31T09:56:32.855-0300] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
